subjects/devops/crud-master/audit/README.md

```diff
@@ -0,0 +1,91 @@
+#### Functional
+
+##### Before we start, let's check we have the necessary tools to perform the tests. You should make sure VirtualBox, Vagrant and Postman are installed, if not you should install them.
+
+#### VM Configuration
+
+##### In the directory of the project run `vagrant up --provider virtualbox` and then run `vagrant status`.
+
+###### Can you confirm that the three VMs (`gateway-vm`, `inventory-vm` and `billing-vm`) are up and running?
+
+##### Locate the `.env` file in the root of the project, run `cat .env`:
```

Comment:
ask to check if they are any credentials in the source code

Reply:
done